bin/site

```diff
@@ -850,6 +854,87 @@
           (error! "Server response:" status (str/trim body))
           (message! "Request id:" (get headers "site-request-id")))))))
 
+(defn infer-content-type
```

Comment:
I've replaced the old one with mine as it supports more types. The main difference is mine will default to octet-stream as per https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types rather than erroring and quitting